Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM ubuntu:latest
+
+ENV config_dir=/root/.chiadog/config.yaml
+ENV TZ=UTC
+
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
```

Comment:
Is this necessary considering this is also executed inside `entrypoint.sh` ?

Reply:
This was imported as-is, I'd assume the intent was to have a consistent UTC TZ when none is provided. But, since UTC is already the unconfigured default this shouldn't be necessary. Will simplify & test.

Reply:
Tests show we don't really need to do any of this mucking, just having the right TZ in the env is enough to get chiadog on the right timezone. So I removed all of it.